src/Gaffer/SplinePlug.cpp

```diff
@@ -106,45 +99,48 @@ bool SplinePlug<T>::acceptsChild( const GraphComponent *potentialChild ) const
 template<typename T>
 PlugPtr SplinePlug<T>::createCounterpart( const std::string &name, Direction direction ) const
 {
-	Ptr result = new SplinePlug<T>( name, direction, defaultValue(), getFlags() );
-	result->setValue( getValue() );
+	Ptr result = new SplinePlug<T>( name, direction, m_defaultPoints, m_defaultInterpolation, getFlags() );
+	typename T::PointContainer points;
+	typename SplinePlug<T>::Interpolation interpolation;
+	getValue( points, interpolation );
```

Comment:
Hmm, `createCounterpart` isn't expected to transfer the current value, just to create a fresh plug with the same defaults. I see you're just mirroring what was there already, but I don't recall a reason why SplinePlug should have been different. Could you try pulling out that `setValue()` call, perhaps in a separate, preceding commit?